### ISSUE TITLE
Brownfield React Native App and WKWebView

### ISSUE THREAD

**Issue by flexaddicted** [datatheorem/TrustKit#191]
Hi all,

I'm working on a brownfield app where I have a native iOS shell where RN components are injected into it.

As a requirement I need to implement Certificate Pinning. TrustKit seems to work well with React Native (setting `kTSKSwizzleNetworkDelegates` as `true`). In particular, I've used a similar configuration:

```
let trustKitConfig: [String: Any] = [
    kTSKSwizzleNetworkDelegates: true,
    kTSKPinnedDomains: [
        "myDomain": [
            kTSKDisableDefaultReportUri: true,
            kTSKEnforcePinning: true,
            kTSKIncludeSubdomains: true,
            kTSKPublicKeyHashes: [
                "pin",
                "backupPin"
            ],
            kTSKPublicKeyAlgorithms: [kTSKAlgorithmRsa2048]
        ]
    ]]

TrustKit.initSharedInstance(withConfiguration: trustKitConfig)
```

Along with React Native I'm using `WKWebView` for specific parts of the application.

My question is: Should I rely on method swizzling for `WKWebView` or do I need to implement the specific delegate like the following?

```
func webView(_ webView: WKWebView, didReceive challenge: URLAuthenticationChallenge, completionHandler: @escaping (URLSession.AuthChallengeDisposition, URLCredential?) -> Void) {
    
    let pinningValidator = TrustKit.sharedInstance().pinningValidator
    // Pass the authentication challenge to the validator;
    // if the validation fails, the connection will be blocked
    if !pinningValidator.handle(challenge, completionHandler: completionHandler) {
        completionHandler(.performDefaultHandling, nil)
    }
}
```

In other words, is it necessary to implement the above delegate? Does method swizzling take care for me this scenario?

Thank you, Lorenzo

**Comment by flexaddicted** [comment]
@nabla-c0d3 Thank you for the reply.
